app/facades/song_facade.rb

```diff
@@ -5,7 +5,9 @@ def initialize(token)
 
   def current_song
     data = service.currently_playing
-    Song.new(data)
+    if data[:is_playing]
```

Comment:
Only creates the song if a song is playing. Maybe there's a better way to do this, but otherwise the data to create the song is empty and so the song is nil